api/python/quilt3/packages.py

```diff
@@ -1333,13 +1336,20 @@ def push(self, name, registry=None, dest=None, message=None, selector_fn=None, *
                 push the local file to s3 (instead of pushing all data to the destination bucket).
             %(workflow)s
             force: skip the top hash check and overwrite any existing package
+            dedupe: don't push if the top hash matches the existing package top hash; return the current package
```

Comment:
I guess I'm OK with this if you feel it's Pythonic but is it wise to change the return type based on a param? Shouldn't we always return `None`? Or is the idea that the return value is how users know that they didn't push? I guess that's OK but it seems weird to me.